sample_apps/run_example.sh

```diff
@@ -674,12 +680,6 @@ function rm_non_git_files() {
         Islet_lib_exists=1
     fi
 
-    echo "${Me}: Delete all files not tracked by git"
-
-    # shellcheck disable=SC2046
-    run_cmd rm -rf $(git ls-files . --exclude-standard --others)
```

Comment:
Delete this `rm`. Now we will only `rm` on L684 those files that are registered in `.gitignore` file.